src/css/FooterTabs.css

```diff
@@ -0,0 +1,26 @@
+ul {
+	list-style-type: none;
+}
```

Comment:
I learned semi-recently, that you can just do `list-style` instead of `list-style-type`.